.github/workflows/e2e.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v3
```

Comment:
Last week they released the V4, since we are updating it, why not to the last one?

Reply:
Because when I did this PR `v4` was not released yet. Solved in [#b7af1b8](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/172/commits/b7af1b81256a6bed680349cf1c3a3526141fa8bb)